packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -344,7 +344,12 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   const ClearedStatusCheck = function* (orderId) {
     const order: BSTransactionType = yield call(api.getPaymentById, orderId)
 
-    if (order.state === 'CLEARED' || order.state === 'COMPLETE' || order.state === 'FAILED') {
+    if (
+      order.state === 'CLEARED' ||
```

Comment:
Why not use an enum for this states?

Reply:
this was a quick fix, but yeah I do agree. I personally force to use enums all around app :)